app/src/main/res/values/styles.xml

```diff
@@ -24,24 +24,31 @@
     </style>
 
     <style name="Text.Title">
-        <item name="android:color">@color/dolphin_gray</item>
+        <item name="android:textColor">@color/dolphin_gray</item>
         <item name="android:textSize">16sp</item>
         <item name="android:textStyle">bold</item>
     </style>
 
     <style name="Text.Text">
-        <item name="android:color">@color/almost_black</item>
+        <item name="android:textColor">@color/almost_black</item>
         <item name="android:textSize">14sp</item>
     </style>
 
     <style name="Text.Text.Bold">
-        <item name="android:color">@color/almost_black</item>
+        <item name="android:textColor">@color/almost_black</item>
```

Comment:
You don't need to include the attributes that are available in the parent. Same goes for other styles edited/introduced in this pull request.